### ISSUE TITLE
Docker Antithesis build failing on CI

### ISSUE THREAD

**Issue by jimmygchen** [sigp/lighthouse#4215]
## Description

The docker build for antithesis has been failing for a while. I think this could be caused by OOM on the GitHub runner, as I was able to get this to pass on a larger runner instance:

https://github.com/sigp/lighthouse/actions/runs/4750121794/jobs/8438019064

We could switch over to one of the self-hosted runners once the following PR is merged: https://github.com/sigp/lighthouse/pull/4115

**Comment by jimmygchen** [comment]
Workflow no longer needed and removed in https://github.com/sigp/lighthouse/pull/4642.
